src/components/Header/components/LoginButton.tsx

```diff
@@ -42,32 +48,39 @@ const LoginButton = chakra(
       }
     }, [isLoginOpen, patronName]);
 
+    useOutsideClick({
```

Comment:
Yes, that's the desired functionality. I just noticed that this works when you click on something _in_ the `Header` component. When you click on something outside of the iframe, it doesn't work. This might be a Storybook thing. I'll fix in a future PR since I need to refactor this as well.

Reply:
Can confirm that this doesn't work in storybook because the component is rendered in an iframe, so the event handlers don't bubble up past the iframe. If you have other components within the same iframe or app, it'll work.

Reply:
Gotcha! Cool.